idl/feed.proto

```diff
@@ -1,33 +1,33 @@
-syntax = "proto3";
-
-package douyin.feed;
-option go_package = "douyin/feed";
-
-import "user.proto";
-
-message Video {
-  uint32 id = 1;
-  user.User author = 2;
-  string play_url = 3;
-  string cover_url = 4;
-  uint32 favorite_count = 5;
-  uint32 comment_count = 6;
-  bool is_favorite = 7;
-  string title = 8;
-}
-
-message ListFeedRequest {
-  optional string latest_time = 1;
-  optional string token = 2;
-}
-
-message ListFeedResponse {
-  uint32 status_code = 1;
-  optional string status_msg = 2;
-  optional int64 next_time = 3;
-  repeated Video videos = 4;
-}
-
-service FeedService {
-  rpc ListVideos(ListFeedRequest) returns (ListFeedResponse);
-}
+syntax = "proto3";
+
+package douyin.feed;
+option go_package = "douyin/feed";
+
+import "user.proto";
+
+message Video {
+  uint32 id = 1;
+  user.User author = 2;
+  string play_url = 3;
+  string cover_url = 4;
+  uint32 favorite_count = 5;
+  uint32 comment_count = 6;
+  bool is_favorite = 7;
+  string title = 8;
+}
+
+message ListFeedRequest {
+  optional string latest_time = 1; // 限制返回视频的最新投稿时间戳，精确到秒，不填表示当前时间
+  optional uint32 requester_id = 2; // 发送请求的用户的id
```

Comment:
Our RPC request may send to web gateway directly, is the rename REALLY ok???

Reply:
> Our RPC request may send to web gateway directly, is the rename REALLY ok???

It's fine to use a customized request parameter. Just call it properly from api gateway